.github/workflows/deployment.yaml

```diff
@@ -131,13 +130,26 @@ jobs:
           STORAGE_ACCOUNT_KEY=$(az storage account keys list --account-name $STORAGE_ACCOUNT_NAME --query "[0].value" -o tsv)
           STORAGE_SHARE_NAME=phdi${{ env.TF_ENVIRONMENT }}tables
           STORAGE_MOUNT_NAME=phdi${{ env.TF_ENVIRONMENT }}tables
-          HASH_SALT=$(az keyvault secret show --vault-name phdi-${{ env.TF_ENVIRONMENT }}-key-vault -n patient-hash-salt --query "value" -o tsv)
+          HASH_SALT=$(az keyvault secret show --vault-name ${{ env.TF_ENVIRONMENT }}vault${CLIENT_ID:0:8} -n patient-hash-salt --query "value" -o tsv)
           if [[ ! $(az containerapp env show -n ${{ env.TF_ENVIRONMENT }}) ]]; then
             az containerapp env create \
               --name ${{ env.TF_ENVIRONMENT }} \
               --resource-group $RESOURCE_GROUP_NAME \
               --location $LOCATION
-            sleep 30
+            echo "Waiting a minute for environment to provision..."
+            sleep 60
```

Comment:
Yeah, this actually takes about 2.5 minutes by my measurement, and I did add polling, but I figured it's worth having a sleep for a minute anyways since it's unlikely to finish within a minute. But I'm happy to remove if you think it makes sense since I already have the polling working.

Reply:
My vote would be to just start polling right away. This way if there is a change to the process for provisioning environments that improves performance we won't have to come back later and decrease/remove this delay.